docs/source/trainer/artifact_logging.rst

```diff
@@ -0,0 +1,96 @@
+|:hindu_temple:| Artifact Logging
+====================
+
+What is an artifact?
+-----------
+
+An artifact can be thought of as a folder stored in the cloud. This folder can contain any
+files or file URL that are pertinent to a training run. These files are called **assets**
```

Comment:
Hmm, throughout the existing docstrings we refer to an individual artifact as a file. We do not reference asset anywhere.